free-1.md

```diff
@@ -10,41 +10,157 @@ _What need is met by your idea? Who are you helping? What is that person's
+So this project would be for logicians who are not familiar enough with LaTeX
+to feel comfortable using the packages for drawing trees.
+Ideally, this project would eliminate the need to drawing out lines and moving
+text boxes using formatting tools like Google Slides, which is the recommended
+formatting tool in CS 81.
 
-### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+People already use a package in a DSL (LaTeX) to draw tableau trees and trees
+in general.
+But a tableau tree has enough distinguishing characteristics from any tree 
+diagram that a DSL for tableau trees should have a simplified interface.
+In CS 81, the construction of a tableau tree is reduced to two operations:
+stack and split. So I think a DSL can use that technique to build a simpler
+syntax for a DSL for tableau trees.
+A DSL for this would make the task much less cumbersome as hopefully users
+would find the DSL simpler to use than drawing out a tree using lines and text
+boxes.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I have to draw tableau trees for my CS 81 homework and they cannot be hand
+drawn.
+I generally TeX my homework, but I've found the syntax of packages for drawing
+trees to be difficult to understand.
+I imagine that the syntax would be even more daunting for people who do not
+use LaTeX for most of their homework.
+I TeX-ed tableau trees in the latest homework but it required a lot of trial
+and error as well as reading a _lot_ of samples of various tree structures.
+I would be really happy if there was an easier way for me to draw trees for 
+my homework.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Writing logic proofs in math.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Like LaTeX, the programmer would write text to a file which would then be
+compiled to the image of a tree.
+One possible way a program could look is
+```
+(A \or B) \and C \check
+C
+A \or B
+  A
+    branch1
+    branch2
+  B
+```
+which would produce the following tree
+
+![tree](/images/tree.png)
+
+This uses LaTeX syntax, such as backslashes to invoke certain symbols, which
+is familiar to many people who work with math.
+The structure is more easily readable for statements that stack (such as
+C and A \or B) compared to the equivalent tree in many tree packages in LaTeX
+which expects a statement on the next line to be the child of the previous
+statement (so A \or B would be the child of C) which would then draw a line
+between them. 
+This would go against tableau convention.
+The package prooftrees solves this issue but still requires A \or B to be
+indented relative to C, which is harder to read.
+
+For logicians, there should not be a need to specify math mode marked by \$
+since everything should be mathematical symbols.
```

Comment:
I think you adequately show that this is the right way to interact with the problem domain because it is easy for users of LaTex while still logical for the logicians (I made a pun). I question if the "\" syntactic sugar is even required for this language? Definitely no "\$" syntax for sure (if possible)